source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,275 @@
+.. _glossary:
+
+=========
+Glossary
+=========
+
+Machine learning terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Categorical feature
+        A :term:`feature` with a discrete domain. Can be :term:`nominal <Nominal
+        feature>` or :term:`ordinal <Ordinal feature>`.
+
+        **Synonyms:** discrete feature, qualitative feature
+
+    Nominal feature
+        A :term:`categorical feature` without ordering between values. Only
+        equality operation is defined for nominal features.
+
+        **Examples:** a person's gender, color of a car
+
+    Classification
+        A :term:`supervised machine learning problem <Supervised learning>` of
+        assigning :term:`labels <Label>` to :term:`feature vectors <Feature
+        vector>`.
+
+        **Examples:** predict what type of object is on the picture (a dog or a cat?),
+        predict whether or not an email is spam
+
+    Clustering
+        An :term:`unsupervised machine learning problem <Unsupervised learning>`
+        of grouping :term:`feature vectors <Feature vector>` into groups,
+        usually :term:`categorical <Categorical feature>`,
```

Comment:
What does "categorical groups" mean? The attempt to logically connect groups and categorical features seems irrelevant here

Reply:
"grouping into groups" sounds odd

Reply:
An unsupervised machine learning problem of grouping a set of feature vectors in a way that observations in the same group (called a cluster) are more similar to each other than to those in other clusters.

Reply:
Rephrased a bit